### PR TITLE
Initial Setup of g-bot

### PR DESCRIPTION
**Pull request by ayushdevelop**
# Related Issue
- https://github.com/orgs/code-mancers/projects/2/views/1

# Functionalities added

-  Added the initial setup for the bot

-  Packages added 
    - `@types/node`
    - `typescript`
    - `dotenv`
    - `@slack/bolt`

### CODE REVIEW

File: server/.env.sample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SLACK_BOT_TOKEN=xoxb-123456-abcdefghijkl
+SLACK_APP_TOKEN=xapp-123456-abcdefghijkl
+SLACK_SIGNING_SECRET=xapp-123456-abcdefghijkl
```
Comment (SujayPrabhu96) [inline server/.env.sample:3]: Add empty line to avoid red mark

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/.env.sample (hunk 1 above)
Comment (ayushdevelop) [reply server/.env.sample:3]: @SujayPrabhu96 can you please elaborate a little I am not able follow this comment?

Comment (SujayPrabhu96) [reply server/.env.sample:3]: Screenshot 2022-09-02 at 11 36 33 AM

**Review by SujayPrabhu96 — APPROVED** [review]
LGTM

**Pull request merged**